lib/hash-tree.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var crypto = require('crypto');
+var fs = require('fs');
+
+function getFileInfos(fullPath) {
+  var stat = fs.statSync(fullPath);
+
+  if (stat.isFile()) {
+    return [{
+      fullPath: fullPath,
+      mtime: stat.mtime.getTime(),
+      mode: stat.mode,
+      size: stat.size
+    }];
+  } else if (stat.isDirectory()) {
```

Comment:
Is it possible the current directory at fullPath is node_modules and the whole thing should be excluded?

Reply:
I don't believe so, as the entry point into the library is a individual dep.